controller/network/router.go

```diff
@@ -106,6 +107,14 @@ func (entity *Router) HasCapability(capability ctrl_pb.RouterCapability) bool {
 	return entity.Metadata != nil && genext.Contains(entity.Metadata.Capabilities, capability)
 }
 
+func (entity *Router) SupportsRouterLinkMgmt() bool {
+	if entity.VersionInfo == nil {
+		return true
+	}
+	supportsLinkMgmt, err := entity.VersionInfo.HasMinimumVersion("0.32.1")
```

Comment:
0.32.1 seems like it should be a constant? maybe documentation/link to the readers of the future here? feels like it could use some meta to me